exoscale/resource_exoscale_instance_pool.go

```diff
@@ -268,20 +268,24 @@ func resourceInstancePoolUpdate(d *schema.ResourceData, meta interface{}) error
 
 	if d.HasChange("name") {
 		req.Name = d.Get("name").(string)
+		d.SetPartial("name")
```

Comment:
You have to call `d.Partial(true)` before calling `d.SetPartial()`: https://godoc.org/github.com/hashicorp/terraform/helper/schema#ResourceData.SetPartial

Reply:
Thanks